ansible/roles/products/hand-e/server/desktop-icons/tasks/hand-and-glove.yml

```diff
@@ -123,13 +123,28 @@
     glove_driver_value: "{{ glove_driver_type }}"
   when: glove=="shadow_glove"
 
+- name: Install the Shadow System Monitor desktop icon for Teleop
+  include_role:
+    name: products/common/web-gui-icon
+  vars:
+    desktop_icon_png: "system_monitor.png"
+    launch_script: "shadow_launcher_system_monitor_exec.sh"
+    local_website_port_var: '8081'
+    desktop_icon_name: "Shadow System Monitor"
+    desktop_icon_path: "Shadow System Monitor"
```

Comment:
something is not right here, https://github.com/shadow-robot/aurora/blob/f3bda17c1b4394514a0eff399f7d232ad6b9bd9f/ansible/roles/products/common/web-gui-icon/tasks/main.yml#L24

Reply:
Theres no issue. It works as intended as seen here:
![Screenshot from 2022-08-12 09-07-15](https://user-images.githubusercontent.com/86227623/184312329-1d717d31-caeb-4802-8dd3-dd9f963e549c.png)

Reply:
desktop_icon_path is only the name of the icon, the 'order' of how the path is created is bad even though it works

Reply:
Its created this way because its moved later on in the process